integration-libs/cpq-quote/cpq-quote-discount/components/cpq-quote-discount-tbody/cpq-quote.component.scss

```diff
@@ -0,0 +1,10 @@
+.center {
```

Comment:
`*.scss` files should be in `styles` folder and not in the edicated component folder.
Please, more this file and verify whether the styling will be applied accordingly.